### DISCUSSION TITLE
Where does `@csstools/postcss-design-tokens` look for tokens files?

### DISCUSSION THREAD

**Discussion by JGallardo4** [csstools/postcss-plugins#1538] [General]
With the example of a CSS file as follows, supposing I use the plugin at the end of my PostCSS pipeline, will it look for the tokens file in the build output, next to the final CSS file?

```css
@design-tokens url('./tokens.json') format('style-dictionary3');
```

It would be great to be able to specify the location of my tokens file in the plugin options

**Comment by romainmenke** [comment]
The current file is the reference point.

```
- ./your-styles.css
- ./tokens.json
```

----

>  supposing I use the plugin at the end of my PostCSS pipeline

Non-standard and sugary plugins should always be used towards the start of your pipeline, never at the end.

The order should be something like this:
- bundler
- non-standard and syntactic sugar plugins
- transpilers/fallbacks for older browser versions
- minifier

---

> It would be great to be able to specify the location of my tokens file in the plugin options

There is no real use case for that.

You can manipulate the path in `@design-tokens url('./tokens.json')` to target any file, relative to the current CSS file.

I can understand how such a plugin options seems like a neat addition but given that it isn't needed I am not inclined to add such a feature. This would make everything more complicated without adding extra capabilities.

**Reply by JGallardo4** [reply]
Thank you for the info!
